LiteCore/Query/QueryParser.cc

```diff
@@ -1548,7 +1548,9 @@ namespace litecore {
 
     // Writes a call to a Fleece SQL function, including the closing ")".
     void QueryParser::writePropertyGetter(slice fn, Path &&property, const Value *param) {
+        size_t property_size_in = property.size();
         auto &&iType = verifyDbAlias(property);
+        bool property_starts_with_explicit_alias = (property.size() + 1 == property_size_in);
```

Comment:
This is a confusing line that I think warrants a comment.  I had to look through the implementation of verifyDbAlias to see why this would ever be true.